docs/getting-started.md

```diff
@@ -0,0 +1,36 @@
+# Getting started with developing your own Custom Metric Server API
+This is an example of how to vendor this project and setup your own provider.  This example uses [glide](https://github.com/Masterminds/glide) but you can use any other [dependency management tool](https://github.com/golang/go/wiki/PackageManagementTools) of your choice.
+
+## Create your new project:
+
+- `mkdir $GOPATH/src/github.com/your-company/my-custom-metric-server`
+- `cd $GOPATH/src/github.com/your-company/my-custom-metric-server`
+
+## Vendor this repository:
+
+- `glide create`
+- `glide get github.com/kubernetes-incubator/custom-metrics-apiserver`
+
+## Create the entry point from the sample:
+
+- `touch main.go` 
+
+> See the [sample entry point](https://github.com/kubernetes-incubator/custom-metrics-apiserver/blob/master/sample-main.go) for a complete example
+
+## Add the server:
+
+- `mkdir -p pkg/cmd/server`
+- `touch pkg/cmd/server/start.go` 
```

Comment:
I wouldn't put this under `pkg/cmd`.  In fact, I'd put main, and this setup, under `cmd/project-name/...`

Reply:
more or less, yes (start and main don't necessarily have to be separate files, though).